spec/fixtures/sshd_config_debian9

```diff
@@ -0,0 +1,142 @@
+# This file is being maintained by Puppet.
+# DO NOT EDIT
+
+# $OpenBSD: sshd_config,v 1.73 2005/12/06 22:38:28 reyk Exp $
```

Comment:
is this file from a default, vanilla debian 9 install?

Reply:
No, I jsut copy the one from debian 7 and fix the change I made on the code.
So I need to put in the original config file ?

Reply:
yeah, put in the original file from a default install and add the header text to the top. The module is designed such that it uses the default values of the OS.

Reply:
We also need the `ssh_config`